airflow/jobs/triggerer_job.py

```diff
@@ -0,0 +1,380 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import asyncio
+import os
+import signal
+import sys
+import threading
+import time
+from collections import deque
+from typing import Deque, Dict, Set, Tuple, Type
+
+from airflow.compat.asyncio import create_task
+from airflow.configuration import conf
+from airflow.jobs.base_job import BaseJob
+from airflow.models.trigger import Trigger
+from airflow.triggers.base import BaseTrigger, TriggerEvent
+from airflow.typing_compat import TypedDict
+from airflow.utils.log.logging_mixin import LoggingMixin
+from airflow.utils.module_loading import import_string
+
+
+class TriggererJob(BaseJob):
+    """
+    TriggererJob continuously runs active triggers in asyncio, watching
+    for them to fire off their events and then dispatching that information
+    to their dependent tasks/DAGs.
+
+    It runs as two threads:
+     - The main thread does DB calls/checkins
+     - A subthread runs all the async code
+    """
+
+    __mapper_args__ = {'polymorphic_identity': 'TriggererJob'}
+
+    def __init__(self, capacity=None, *args, **kwargs):
+        # Call superclass
+        super().__init__(*args, **kwargs)
+
+        if capacity is None:
+            self.capacity = conf.getint('triggerer', 'default_capacity', fallback=1000)
```

Comment:
```suggestion
            self.capacity = conf.getint('triggerer', 'default_capacity')
```

(Eventually I'll start my crusade against `fallback`, but this will mean 1 less when that time comes)